libraries/classes/Config/SpecialSchemaLinks.php

```diff
@@ -66,7 +66,7 @@ public static function get(): array
     {
         global $cfg;
 
-        $defaultPage = './' . Util::getScriptNameForOption($cfg['DefaultTabTable'], 'table');
+        $defaultPage = './' . Util::getScriptNameForOption($cfg['DefaultTabTable'], 'database');
```

Comment:
```suggestion
        $defaultPage = './' . Util::getScriptNameForOption($cfg['DefaultTabDatabase'], 'database');
```

I concluded this fix is wrong without this change. Any opinions @liviuconcioiu ?
https://github.com/phpmyadmin/phpmyadmin/blob/aeab8a1c4172c46190649a86d404487deac183ee/libraries/classes/Util.php#L1552-L1571